Makefile

```diff
@@ -70,3 +71,12 @@ beats-dashboards:
 .PHONY: docs
 docs:
 	sh libbeat/scripts/build_docs.sh ${PROJECTS}
+
+.PHONY: package
+package:
+	$(MAKE) -C libbeat package-setup
+	$(foreach var,$(BEATS),SNAPSTHOT=$SNAPSHOT $(MAKE) -C $(var) package || exit 1;)
```

Comment:
Side note: I plan to make it in a future PR parallel builds again based on one of the tricks here: http://stackoverflow.com/questions/1681006/parallelization-of-recursive-jobs-in-gnu-make